src/dictionary.js

```diff
@@ -0,0 +1,12 @@
+const dictionary = {
+  IE: {
+    avif: () => false,
+    webp: () => false,
```

Comment:
To have support for the new [JPEG-XL](https://jpeg.org/jpegxl/) image format

```suggestion
    webp: () => false,
    jpegxl: () => false,
```

Reply:
fixed!